modules/administration-guide/attachments/restricted-environment/prepare-restricted-environment.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
I don't know that maintaining a script like this is suitable for the docs repo. If we decide something like this is needed, it should likely live somewhere in the `che-operator` repo.

Reply:
It's what we do with https://www.eclipse.org/che/docs/stable/administration-guide/upgrading-che-7-41-on-openshift/. 

I was reluctant at first to onboard code in the docs repository, but convinced by this: we keep is simple and stupid: the link to the script is easy to maintain: hosted with the docs, no worries about generating a URL with the accurate version.

However, I believe this script deserves to be hosted in another repository for proper (automated) testing of the script. Probably it can be done in a subsequent iteration, as it is critical to deliver a first round for Dev Spaces 3.0.

Reply:
Makes sense to me. In the future, since this script would also potentially be part of automated airgap testing and require maintenance, it will make sense to move it into a regular Che repo.

Reply:
Dreaming out loud: integrate it as a `chectl` / `dsc` command. Something like:

```
chectl airgap:prepare
```

(maybe with subcommands to address also a fully airgapped scenario?)